src/components/header/linkList.js

```diff
@@ -0,0 +1,30 @@
+export const guestLinks = [
```

Comment:
Please, specify all links directly in markup. Or structure it inside an appropriate component. Don't extract it into separate file.